spec/mongoid/attributes/nested_spec.rb

```diff
@@ -204,7 +215,12 @@
     context "when the relation is a referenced in" do
 
       before do
-        Post.accepts_nested_attributes_for :person
+        Post.accepts_nested_attributes_for :person, autosave: false
+      end
+
+      after do
+        Post.send(:undef_method, :person_attributes=)
+        Post.nested_attributes.clear
```

Comment:
Please do not mutate models like this. The test suite defines a ton of models, if there isn't a suitable one in it already please add another one but a model's definition should not be mutated in the test suite like this.

Reply:
The test suite was already mutating the model (see right above). This is just setting the model back to the way it was before.

Reply:
It's bad there too.